desc/grid.py

```diff
@@ -231,6 +231,16 @@ def is_meshgrid(self):
         """
         return self.__dict__.setdefault("_is_meshgrid", False)
 
+    @property
+    def can_fft(self):
```

Comment:
Can we rename this to `can_fft2` or something? In #1405 I added 2 attributes for the poloidal and toroidal parts (since some grids may be able to do an fft in one but not the other)